### PR TITLE
Dev nxtflasher 550

### PR DESCRIPTION
**Pull request by FrikadellenFred**

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -444,6 +444,14 @@ if 'NETX90' in atPickNetxForBuild:
         HBOOTIMAGE_KNOWN_FILES=dict(),
         HBOOTIMAGE_DEFINES=dict(),
         ASIC_TYP='NETX90B')
+
+    set_sip_protection_cookie_com_usp = env_netx90_nodbg_secure.HBootImage(
+        os.path.join('targets', 'disable_security_settings', 'set_sip_protection_cookie.usp'),
```
Comment (dizzydevil) [inline SConstruct:449]: That path (targets...)  is not in the release package, or is it?

File: lua/lua/sipper.lua [hunk 2]
```diff
@@ -44,7 +54,7 @@ function Sipper:verify_usip(tUsipConfigData, strComSipFilePath, strAppSipFilePat
             strCompareSipPath = strAppSipFilePath
             strCompSip = "APP"
         else
-            tResult = false
+            uResult = VERIFY_RESULT_ERROR
```
Comment (dizzydevil) [inline lua/lua/sipper.lua:57]: The for loop continues with the next usip if an error/mismatch occurs. Is this intentional?

File: lua/usip_player.lua [hunk 3]
```diff
@@ -2527,20 +2553,16 @@ elseif tArgs.fCommandGetUidSelected then
         strExecReturnPath
     )
 
-    if not fFinalResult then
```
Comment (dizzydevil) [inline lua/usip_player.lua:2530]: Why remove this?

File: lua/usip_player.lua [hunk 4]
```diff
@@ -2527,20 +2553,16 @@ elseif tArgs.fCommandGetUidSelected then
...
-        os.exit(1)
-    else
-        os.exit(0)
-    end
 
 --------------------------------------------------------------------------
 -- VERIFY CONTENT
 --------------------------------------------------------------------------
-elseif tArgs.fCommandVerifySelected then
+elseif tArgs.fCommandVerifySelected or tArgs.fCommandCheckSIPCookie then
```
Comment (dizzydevil) [inline lua/usip_player.lua:2555]: I'd make a new section for check SIP cookie, most of this block is specific to it anyway.
As a message, I'd use "CHECKING FOR SIP COOKIE"

File: lua/lua/sipper.lua [hunk 5]
```diff
@@ -30,7 +34,13 @@ function Sipper:_init(tLog)
 end
 
 function Sipper:verify_usip(tUsipConfigData, strComSipFilePath, strAppSipFilePath)
-    local tResult = true
+    -- verify the configuration data extracted from an usip file with the content of the COM and APP SIP
+    -- return values:
+    --  0: (RESULT_VERIFY_OK) data verified
+    --  1: (RESULT_VERIFY_NOT_OK) data not verified
+    --  2: (RESULT_VERIFY_ERROR) error while verifying
```
[the remark this answers is not in the document]
Comment (FrikadellenFred) [reply lua/lua/sipper.lua:41]: fixed

**Review by dizzydevil — APPROVED** [review]
see the individual comments.

**Pull request merged**